tools/MigrationDefaultsConstructor/Program.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Execution;
+using Newtonsoft.Json;
+using System.Threading;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration.Models;
+
+namespace MigrationDefaultsConstructor
+{
+    public class Program
+    {
+        private const string c_temporaryDotnetNewMSBuildProjectName = "p";
+
+        static void Main(string[] args)
+        {
+            var dotnetPath = Command.Create("which", new [] { "dotnet" }).CaptureStdOut().Execute().StdOut.Trim();
+            
+            var targetsFile = ProjectRootElement.Open(Path.Combine(Directory.GetCurrentDirectory(), "Microsoft.NETCore.Sdk.targets"));
```

Comment:
Magic, I'll stick a readme with instructions, you just need to put them there yourself. 
This tool should be run infrequently.